ra/fsp/ra/fsp/src/bsp/mcu/ra8m1/bsp_feature_gen.h

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef BSP_FEATURE_GEN_H
+#define BSP_FEATURE_GEN_H
+
+/***********************************************************************************************************************
+ * Includes   <System Includes> , "Project Includes"
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Macro definitions
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Typedef definitions
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Exported global variables (to be accessed by other files)
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Private global variables and functions
+ **********************************************************************************************************************/
+// *UNCRUSTIFY-ON*
```

Comment:
If you delete it, please delete it in a balanced manner.

In the original code, this line was paired with
```
// *UNCRUSTIFY-OFF*
```

Reply:
>There are the following differences when compared with fsp, but is it necessary to delete them?
It's mistake when migrate to FSPv5.3.0. It will need for GPT. We will update it in this week.

Reply:
I confirmed that it was fixed.